src/AppBundle/Controller/TopEditsController.php

```diff
@@ -65,11 +65,22 @@ public function indexAction(Request $request)
 
         $project = ProjectRepository::getProject($projectName, $this->container);
 
+        // Default values for the variables to keep the template happy
+        $domain = null;
+        $namespaces = null;
+
+        // If the project exists, actually populate the values
+        if ($project->exists()) {
+            $domain = $project->getDomain();
+            $namespaces = $project->getNamespaces();
+        }
+
         return $this->render('topedits/index.html.twig', [
             'xtPageTitle' => 'tool-topedits',
             'xtSubtitle' => 'tool-topedits-desc',
             'xtPage' => 'topedits',
-            'project' => $project,
+            'domain' => $domain,
+            'namespaces' => $namespaces,
```

Comment:
Indeed.  But in this case, it also allows us to set $domain and $namespaces to default values - twig templates don't have very good error handling from what I can tell.